oneoffs/joseki/public/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
I can't leave a comment on favicon.ico, if it's the default react favicon please remove it

Reply:
nope, i made a new one :)